README.md

```diff
@@ -47,4 +47,7 @@ We hope you find this course informative and useful! If you have any questions o
 
 Happy learning and contributing!
 
+🤝 Contributing to this Repository
+We encourage you to contribute to OpenSauced! All contributors are required to abide by our [Code of Conduct](https://github.com/open-sauced/.github/blob/main/CODE_OF_CONDUCT.md). Please check out the [Contributing guide](https://docs.opensauced.pizza/contributing/introduction-to-contributing/) for guidelines about how to proceed with your contribution.
```

Comment:
[proofreading] The beginning of this paragraph is off too a good start, but consider making it more engaging by adding a Call To Action (e.g., mentioning specific ways people can contribute to the course). Doing so would encourage people to start contributing. 
```suggestion
See things on this course that needs improvement? Feel like it needs a lesson on a specific topic? Share your contribution! 😊All contributors are required to abide by our [Code of Conduct](https://github.com/open-sauced/.github/blob/main/CODE_OF_CONDUCT.md). Please check out the [Contributing guide](https://docs.opensauced.pizza/contributing/introduction-to-contributing/) for guidelines about how to proceed with your contribution. We can’t wait to hear from you! 😊
```